profiler/src/ProfilerEngine/Directory.Build.targets

```diff
@@ -29,52 +29,44 @@
+          <DDPlatform Condition=" '$(PlatformShortName)' != '' AND $([MSBuild]::IsOSPlatform('Windows'))">win-$(PlatformShortName)</DDPlatform>
+          <DDPlatform Condition=" '$(PlatformShortName)' == '' AND $([MSBuild]::IsOSPlatform('Windows'))">win-unknown</DDPlatform>
+          <DDPlatform Condition=" '$(Platform)' != '' AND $([MSBuild]::IsOSPlatform('Linux')) ">linux-$(Platform)</DDPlatform>
+          <DDPlatform Condition=" '$(Platform)' == '' AND $([MSBuild]::IsOSPlatform('Linux')) ">linux-unknown</DDPlatform>
+          
+          <DeployDirectoryBase >$(TopLevelDeployDirectoryBase)\$(DDPlatform)</DeployDirectoryBase>
         </PropertyGroup>
 
         <PropertyGroup>
-            <!--
-            Note that the top-level Directory.Build.props defines a property "ConfigBasedRelativeOutputPath" that
-            combines $(Configuration) with the Platform moniker just like below. However we cannot use it, because
-            it uses the Platform moniker of the current build, whereas here we need all the different relevant Platform monikers.
-            So, if we ever renamed a platform moniker (it happened in the past), we would need to update the lines below accordingly.
-            -->
-            <NativeWindowsX86DeployFilesSrcDir>$(BaseOutputPath)\$(Configuration)-x86\profiler\src\ProfilerEngine\Datadog.Profiler.Native.Windows</NativeWindowsX86DeployFilesSrcDir>
-            <NativeWindowsX64DeployFilesSrcDir>$(BaseOutputPath)\$(Configuration)-x64\profiler\src\ProfilerEngine\Datadog.Profiler.Native.Windows</NativeWindowsX64DeployFilesSrcDir>
-
-            <NativeLinuxX86DeployFilesSrcDir>$(BaseOutputPath)\$(Configuration)-x86\profiler\src\ProfilerEngine\Datadog.Profiler.Native.Linux</NativeLinuxX86DeployFilesSrcDir>
-            <NativeLinuxX64DeployFilesSrcDir>$(BaseOutputPath)\$(Configuration)-x64\profiler\src\ProfilerEngine\Datadog.Profiler.Native.Linux</NativeLinuxX64DeployFilesSrcDir>
+            <NativeWindowsDeployFilesSrcDir>$(BaseOutputPath)\$(ConfigBasedRelativeOutputPath)\profiler\src\ProfilerEngine\Datadog.Profiler.Native.Windows</NativeWindowsDeployFilesSrcDir>
+            <NativeLinuxDeployFilesSrcDir>$(BaseOutputPath)\$(ConfigBasedRelativeOutputPath)\profiler\src\ProfilerEngine\Datadog.Profiler.Native.Linux</NativeLinuxDeployFilesSrcDir>
 
         </PropertyGroup>
 
         <ItemGroup>
-            <NativeX86DeployFiles Include="$(NativeWindowsX86DeployFilesSrcDir)\*.dll" />
-            <NativeX86DeployFiles Include="$(NativeWindowsX86DeployFilesSrcDir)\*.pdb" />
-            <NativeX86DeployFiles Include="$(NativeLinuxX86DeployFilesSrcDir)\*.so" />
-
-            <NativeX64DeployFiles Include="$(NativeWindowsX64DeployFilesSrcDir)\*.dll" />
-            <NativeX64DeployFiles Include="$(NativeWindowsX64DeployFilesSrcDir)\*.pdb" />
-            <NativeX64DeployFiles Include="$(NativeLinuxX64DeployFilesSrcDir)\*.so" />
+            <NativeDeployFiles Include="$(NativeWindowsDeployFilesSrcDir)\*.dll" />
```

Comment:
Also, the simple answer is that `NativeWindowsDeployFilesSrcDir` is different for different architectures 🙂

Reply:
I have to admit that I don't see NativeWindowsDeployFilesSrcDir set. 
However, is it the "source" from which the files will be copied (could be different because generated during the build)?
Here, I'm talking about the destination in DDProf-Deploy

Reply:
> I have to admit that I don't see NativeWindowsDeployFilesSrcDir set.
It's set on line 46, on your previous comment, and includes `Release-x64` etc in the path.

> Here, I'm talking about the destination in DDProf-Deploy
The destination is given by `DeployDirectoryBase`, which is `DDProf-Deploy/win-x64` and `DDProf-Deploy/win-x86` respectively.

On _linux_ we currently still build to the root, as we only build one architecture. We would need to update the CMakefile etc to account for that if we start building for multiple arches at once on Linux. But that's a current issue too, so there's no change in that